index.js

```diff
@@ -14,11 +14,17 @@
 //. [![CircleCI](https://img.shields.io/circleci/project/github/sanctuary-js/sanctuary/master.svg)](https://circleci.com/gh/sanctuary-js/sanctuary/tree/master)
 //. [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/sanctuary-js/sanctuary)
 //.
-//. Sanctuary is a functional programming library inspired by Haskell
-//. and PureScript. It's stricter and more opinionated than [Ramda][],
-//. and provides a similar suite of functions.
+//. Sanctuary is a functional programming library inspired by Haskell and
+//. PureScript. It provides a similar suite of functions as [Ramda][],
+//. but in a stricter and more opinionated way, ensuring errors are
+//. properly handled.
+//. Sanctuary promotes programs composed of simple, pure functions.
+//. Such programs are easy to test, comprehend, and maintain, and a
+//. pleasure to write.
+//. Sanctuary provides two [Fantasy Land][]-compatible data types,
+//. Maybe and Either, which mean that even functions which may fail
+//. — such as `head` — are composable.
```

Comment:
One thing that's unclear is whether the text above is meant to be one paragraph or three. Since we're using Markdown, paragraphs must be separated by empty lines. If you intend the text above to be three paragraphs, please add empty lines to separate the paragraphs. If you intend the text above to be a single paragraph, please make this clear by having the sentences flow.